README.md

```diff
@@ -1,3 +1,5 @@
 # iOS Codegen Template
 
-This is the template for using Swift-based scripting to generate iOS code. Please see our full instructions at [https://www.apollographql.com/docs/ios/swift-scripting/](https://www.apollographql.com/docs/ios/swift-scripting/). 
\ No newline at end of file
+This is the template for using Swift-based scripting to generate iOS code. Please see our full instructions at [https://www.apollographql.com/docs/ios/swift-scripting/](https://www.apollographql.com/docs/ios/swift-scripting/). 
+
+Requires a minimum version of 0.49.0 of Apollo.
```

Comment:
We've been updating the version in the `Package.swift` to serve as the minimum since the design here is to have developers download and use the template directly rather than copying things out of the files. 

Rather than having that information in two places, would it work to have this point to the `Package.swift` file for people who are copying things out of the files?

Reply:
I don't think that would be obvious enough. At least not for me, to have given up on this and moved back to the normal CLI.

This through me from the docs

> Important! Since a particular version of code generation is tied to a particular version of the SDK, you need to make sure that the dependencies section of Package.swift is set to grab the same version of the apollo-ios library you're using in your main application:

That reads, change the version in Package.swift to match your version of the library. That's exactly what I did. Possibly a comment in the Package.swift file would have made a minimum version more obvious, but it needs to be stated explicitly, or IMO the docs should be changed to mention the minimum version.

Reply:
Ah, true. That is still important, I'll think about a better way to word all this though.